src/main/java/fi/metatavu/dcfb/server/rest/LocationsApiImpl.java

```diff
@@ -0,0 +1,206 @@
+package fi.metatavu.dcfb.server.rest;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.UUID;
+
+import javax.ejb.Stateful;
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+import javax.ws.rs.core.Response;
+
+import org.apache.commons.lang3.ArrayUtils;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.apache.commons.lang3.ArrayUtils'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=11&issues=AWQuXfVvYwZU9vvrs6iz&open=AWQuXfVvYwZU9vvrs6iz)

[ISSUE_KEY]: AWQuXfVvYwZU9vvrs6iz